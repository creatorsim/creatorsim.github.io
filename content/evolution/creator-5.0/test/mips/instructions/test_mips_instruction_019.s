
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li $t0, 0x22346
    li $t1, 4567
    li $t2, -9486

    andi $t3, $t0, 0x455
    andi $t4, $t1, 0x222
    andi $t5, $t2, 0xFFF
  
