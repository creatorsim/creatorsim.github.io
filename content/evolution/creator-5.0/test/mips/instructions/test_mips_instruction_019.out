
cr[PC]:0x28; ir[1,at]:0xffffdaf2; ir[8,t0]:0x22346; ir[9,t1]:0x11d7; ir[10,t2]:0xffffdaf2; ir[11,t3]:0x44; ir[12,t4]:0x2; ir[13,t5]:0xaf2; keyboard[0x0]:''; display[0x0]:''; 

